### DISCUSSION TITLE
Again about RADIATIVE HEAT FLUX GAS and INTEGRATED INTENSITY

### DISCUSSION THREAD

**Discussion by mst-parkhachev** [firemodels/fds#12715] [General]
Continuing the topic
https://groups.google.com/g/fds-smv/c/C9y44pD7758

Dear Colleagues!
I wrote a short document and attached it
[About Heat Flux.pdf](https://github.com/firemodels/fds/files/14815972/About.Heat.Flux.pdf)

1. Please see Chapter 1 and confirm whether I understand correctly which integrals define RADIATIVE HEAT FLUX GAS and INTEGRATED INTENSITY
2. It seems to me that this document (Chapter 3) also shows that using INTEGRATED INTENSITY to study human safety is too conservative (since it integrates background radiation).

Thank you!

**Comment by mcgratta** [comment]
In the GoogleGroups post, I said that `INTEGRATE INTENSITY` is approximately 4 times the directional heat flux averaged over the unit sphere. Thus, we do not recommend that `INTEGRATED INTENSITY` should be used in this way.
